docs/pages/development/raster_v1.rst

```diff
@@ -0,0 +1,253 @@
+
+``complex``
+
+This field must be true if present, and when not present it is implicitly false. If set, this band has two identical data types - a “real” one and an “imaginary” one, and each of these data types is of the size specified - such that the entire band takes up twice the specified size.
+
+``noData``
+
+Specifies which value should not be read literally, but instead should be interpreted as meaning “no data” eg ``0``, ``-1``, ``0xffffffff``, ``+Inf``. If not present, there is no single value that means “no data”.
+
+``hasMask``
+
+This field must be true if present, and when not present it is implicitly false. If true, this band is accompanied by a 1-bit-integer band which controls whether this band is visible/invisible (or data/no-data) at a particular pixel. This mask band is not otherwise shown in the schema.
+
+
+``unsigned`` This field must be true if present, when not present it is implicitly false. When present, it specifies that an integer band should be treated as unsigned, and when not present, the integer band is interpreted as signed. This property only applies to band with a data type of "integer".
+
+
+``meta/crs.wkt``
+^^^^^^^^^^^^^^^^
+
+This is the Coordinate Reference System that is common to all the tiles, stored in the `Well Known Text format <well_known_text_format_>`_
+
+``meta/band/1/rat.xml``
+^^^^^^^^^^^^^^^^^^^^^^^
+
+This is indexed starting at one - ``band/1/rat.xml`` is for the first band, ``band/2/rat.xml`` is for the second band, and so on. Only certain bands will have this metadata attached.
+
+This XML contains the column headings of the `Raster Attribute Table <rat_>`_ associated with this band, if there is one. This information is extracted from the `Persistent Auxiliary Metadata <pam_>`_ (PAM) files (the ``.aux.xml`` files) associated with the tiles, if there are any.
+
+If a particular band has a raster-attribute-table, it must have the same raster attribute table for every tile: that is, every instance of that raster attribute table attached to each tile should have the same number of fields with the same field definitions. However, the rows of the raster attribute table may vary from tile to tile. This is a continuation of the principle that every tile should have the same schema, but the actual data of each tile may vary to any extent.
+
+``meta/band/1/categories.json``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This is indexed starting at one - ``band/1/categories.json`` is for the first band, ``band/2/categories.json`` is for the second band, and so on. Only certain bands will have this metadata attached.
+
+This JSON object describes a mapping from each possible value that the band can take, to a freeform text description of what that value means. As a simple example,
+it could be ``{"1": "Land", "2": "Water"}``. This information is extracted from the `Persistent Auxiliary Metadata <pam_>`_ (PAM) files (the ``.aux.xml`` files)associated with the tiles, if there are any.
```

Comment:
```suggestion
it could be ``{"1": "Land", "2": "Water"}``. This information is extracted from the `Persistent Auxiliary Metadata <pam_>`_ (PAM) files (the ``.aux.xml`` files) associated with the tiles, if there are any.
```